inference-engine/src/gna_plugin/transformations/swap_input_matmul_gna.hpp

```diff
@@ -16,4 +16,16 @@ class SwapInputMatMul: public ngraph::pass::MatcherPass {
     NGRAPH_RTTI_DECLARATION;
     SwapInputMatMul();
 };
+
```

Comment:
At least transformation tests for new transformations are required

Reply:
removed those transformations, they needed to fix another issue adn added here by mistake